templates/email_report_violation.html

```diff
@@ -0,0 +1,13 @@
+<p>Hello Admin,</p>
+<br>
+<p>A violation of code of conduct has been reported by {{ reporter_name }} in their mentorship relation.</p>
+<br>
```

Comment:
Send usernames and userids/useremail both for reporter and commenter here.

Reply:
Done